tests/collection-requirements.yml

```
@@ -0,0 +1,5 @@
+# SPDX-License-Identifier: MIT
+---
+collections:
+  - ansible.windows
+  - fedora.linux_system_roles
```

```suggestion
  - name: fedora.linux_system_roles
```
https://linux-system-roles.github.io/documentation/role-requirements.html